src/App.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import Home from 'pages/Home';
 import GlobalStyles from 'styles/GlobalStyles';
+import MultiSelect from './components/atoms/MultiSelect';
 
 function App(): JSX.Element {
   return (
     <>
       <GlobalStyles />
-      <Home />
+      <MultiSelect isMulti />
```

Comment:
In this component we need props like options and callback with values

Reply:
Props like option, okey I can agree, but if we use redux in the future, probably any callback won't be neccessary to pull out. What do you think ?